GATEWAY/src/pages/login/login.ts

```diff
@@ -25,6 +25,7 @@ export class LoginPage {
    * @param {string} user - username
    * @param {string} host - api host address
    * @param {number} port - mqtt port number
+   * @param {boolean} remember - if the login credentials should be remembered
```

Comment:
Missing from the parameter description
It is used as a parameter in the connectToServer method, but was not described